generator/templates/api.h

```diff
@@ -109,6 +109,8 @@ typedef struct {{c_prefix}}ChainedStructOut {
 extern "C" {
 #endif
 
+typedef void (*{{c_prefix}}Proc)(void);
```

Comment:
How about keeping the function pointer in the dawn.json

Reply:
cause it is common for all APIs, let's define it in the template